### DISCUSSION TITLE
Didn't find Device files....

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#6137] [Q&A]
Hi there, as admin i see the "Device Groups file folder". 
But for my user, where is there right to set this ?
I don't find it :(
Regards

**Comment by si458** [comment]
you need to show screenshots what ur talking about?

**Reply by simpsonetti** [reply]
Okay, Monday i will make a screenshot. 
But as a description for that topic: when i create an Device Group, i have automaticly an Folder with the same name as the device group in the file area. 
User, i give "file access" they have their private files folder, but i don't see where to configure it for device group file access.

**Comment by simpsonetti** [comment]
SO, here are the screenshots of my device, but without File-acces to the devices groups. 
![file_access_2](https://github.com/Ylianst/MeshCentral/assets/57707812/e2328904-966b-47fa-b81d-2615d7e3c4ba)
![file_access_1](https://github.com/Ylianst/MeshCentral/assets/57707812/e011fbd6-933f-4a14-89b5-fe8be5caca5c)

**Reply by si458** [reply]
you need to share what permissions you have set for the user and groups etc

as im still not too sure what you are asking?

**Comment by si458** [comment]
oh i think i understand what you mean
when u create a group and set permissions, the user cant see the groups 'my files' listed
you need to add the 'server files' permission when you add a group to a user

**Reply by simpsonetti** [reply]
hmm, okay, i thougth i had done this. I check this tomorrow.
